### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by gonfva** [aws-cloudformation/aws-cloudformation-rpdk-go-plugin#84]
On macOS Catalina, when trying to initialise a project, I get an Unhandled exception
```
$ cfn init
Initializing new project
What's the name of your resource type?
(Organization::Service::Resource)
>> BCL::Platform::Database
One language plugin found, defaulting to go
File already exists, not overwriting '/Users/gonfva/code/coreos-cfn/bcl-platform-database.json'
Enter the GO Import path
>> database
=== Unhandled exception ===
Please report this issue to the team.
Issue tracker: https://github.com/aws-cloudformation/aws-cloudformation-rpdk/issues
Please include the log file 'rpdk.log'
```
The last attempt that appears in rpdk.log shows the following
```
[2019-11-21T22:07:25Z] DEBUG    - Logging set up successfully
[2019-11-21T22:07:25Z] DEBUG    - Running init: Namespace(command=<function init at 0x105b424d0>, force=False, subparser_name='init', verbose=0, version=False)
[2019-11-21T22:07:25Z] DEBUG    - Root directory: /Users/gonfva/code/coreos-cfn
[2019-11-21T22:07:25Z] WARNING  - Initializing new project
[2019-11-21T22:07:25Z] DEBUG    - Loading project file '/Users/gonfva/code/coreos-cfn/.rpdk-config'
[2019-11-21T22:07:39Z] DEBUG    - Resource type identifier: BCL::Platform::Database
[2019-11-21T22:07:39Z] WARNING  - One language plugin found, defaulting to go
[2019-11-21T22:07:39Z] DEBUG    - Language plugin: go
[2019-11-21T22:07:39Z] WARNING  - File already exists, not overwriting '/Users/gonfva/code/coreos-cfn/bcl-platform-database.json'
[2019-11-21T22:07:39Z] DEBUG    - Init started
[2019-11-21T22:07:46Z] DEBUG    - Writing .gitignore: /Users/gonfva/code/coreos-cfn/.gitignore
[2019-11-21T22:07:46Z] DEBUG    - Unhandled exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/cli.py", line 98, in main
    args.command(args)
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/init.py", line 153, in wrapper
    function(args)
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/init.py", line 143, in init
    project.init(type_name, language)
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/project.py", line 181, in init
    self._plugin.init(self)
  File "/usr/local/lib/python3.7/site-packages/rpdk/go/codegen.py", line 59, in init
    contents = resource_stream(__name__, "data/go.gitignore").read()
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/data_loaders.py", line 32, in resource_stream
    f = pkg_resources.resource_stream(package_name, resource_name)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1151, in resource_stream
    self, resource_name
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1613, in get_resource_stream
    return open(self._fn(self.module_path, resource_name), 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.7/site-packages/rpdk/go/data/go.gitignore'
```
I can confirm that in rpdk/go there is no data directory
```
$ ls -a /usr/local/lib/python3.7/site-packages/rpdk/go/
.		..		__init__.py	__pycache__	codegen.py	resolver.py	utils.py
```
Installation was using pip3
```
$ pip3 install --no-cache-dir --upgrade --force-reinstall cloudformation-cli-go-plugin
Collecting cloudformation-cli-go-plugin
  Downloading https://files.pythonhosted.org/packages/88/23/3a257892672da846fcba478ee6c94200d3e06ba863659b6780e0e3fa8c3a/cloudformation_cli_go_plugin-0.1-py3-none-any.whl
Collecting cloudformation-cli<0.2,>=0.1
  Downloading https://files.pythonhosted.org/packages/08/25/1dcd440642abaa4eb64e04feee15e26f2dca6b3eb18f4cb941726352a6f4/cloudformation_cli-0.1.1-py3-none-any.whl
```
Thank you

**Comment by gonfva** [comment]
I've been trying to trace the issue but my experience with python is extremely reduced. 
However, I've noticed that this line
https://github.com/aws-cloudformation/aws-cloudformation-rpdk-go-plugin/blob/master/setup.py#L36
has a comment "package_data -> use MANIFEST.in instead"
but I think there is no MANIFEST.in.

**Comment by kadrach** [comment]
The manifest file is indeed missing!

If you are comfortable building/installing python packages from source, you can manually create a `MANIFEST.in` and install. The patch below adds the data and templates folder, with these I can successfully `init` a project.

```diff
diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 0000000..e77ad1d
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1,2 @@
+graft python/rpdk/go/data
+graft python/rpdk/go/templates
```

**Comment by gonfva** [comment]
I can confirm that using that MANIFEST.in and installing from source, I can init the project correctly. 

Thank you!!!

**Comment by jotompki** [comment]
This looks to be fixed in the latest release [v0.1.1](https://github.com/aws-cloudformation/aws-cloudformation-rpdk-go-plugin/releases/tag/v0.1.1).
